src/cli/MacCodacyCli.ts

```diff
@@ -96,9 +96,11 @@ export class MacCodacyCli extends CodacyCli {
   }
 
   public async update(): Promise<void> {
-    const command = `${this.getCliCommand()} update`
+    const updateCommand = `${this.getCliCommand()} update`
+    const resetCommand = `${this.getCliCommand()} config reset`
```

Comment:
this one I think needs the repo and account token information I think (if available, of course)